CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.4.0 (April 6, 2021)
+
+FEATURES:
+
+- **New resource** `hcp_vault_cluster` (#97)
+- **New resource** `hcp_vault_cluster_admin_token` (#97)
+
+IMPROVEMENTS:
+
+- all: General chores (#82)
```

Comment:
Oh sorry, should have maybe made it more clear, but I thought we'd break these out into individual items:
```
FIXES:
- all: Ensure context is being passed for all HCP API calls (#82)

IMPROVEMENTS
resource/aws_network_peering: Update comments, docs, and messages to use correct capitalization for network peering (#82)
all: Log import ID used when an import fails due to parsing (#82)
all: Add comment to clarify that Links can be sent in API requests (#82)
resource/aws_network_peering: Update peering wait function to use helper (#82)
```